### PR TITLE
Add Then-Framework and fixed some issues

### PR DESCRIPTION
**Pull request by dungi**
Switched dark and lightColor-Values
Makes SwiftDate optional (very long build-time and not needed in all projects by default)
Fixes swiftlint-warning for functions ending with 'print()'

### CODE REVIEW

File: Sources/Code/Record.swift [hunk 1]
```diff
@@ -6,48 +6,48 @@
 //  Copyright © 2017 Jamit Labs GmbH. All rights reserved.
 //
 
-// NOTE: If you're using Realm, uncomment the code below and subclass `Record` instead of `Object` for your Realm objects. Also remove this files entry from .swiftlint.yml.
+//  NOTE: If you're using Realm, uncomment the code below and subclass `Record` instead of `Object` for your Realm objects. Also remove this files entry from .swiftlint.yml.
```
Comment (Jeehut) [inline Sources/Code/Record.swift]: Why did you uncomment the entire class? Please revert.

**Review by Jeehut — CHANGES_REQUESTED** [review]
Thanks you for the PR, looks nearly good. Just one thing, see my review comment.

File: Sources/Code/Record.swift (hunk 1 above)
Comment (Jeehut) [reply Sources/Code/Record.swift]: Ah, I see now, you've unchecked the target. Please update the comment then to reflect your change. Also remove the duplicate whitespace after `//`.

**Comment by dungi** [comment]
Is this better now vong reflection?

**Review by Jeehut — APPROVED** [review]

**Pull request merged**